wordpress-mobile/WordPress-iOS
# ULS Get Started: show Terms of Service

Ref: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/404
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/414

This updates Auth to show the Terms of Service cell in the Get Started view.

To test:
- Enable the `unifiedWordPress` Feature Flag.
- Log In > Continue with WordPress.com.
- On the Get Started view, tap the Terms of Service.
- Verify the Terms of Service page is opened. 

| ![get_started_tos](https://user-images.githubusercontent.com/1816888/91355906-11f6ce80-e7ac-11ea-8d7e-d9a8638dbbad.png) | ![tos_page](https://user-images.githubusercontent.com/1816888/91355919-16bb8280-e7ac-11ea-98dc-79e1d7dfdc61.png) |
|--------|-------|

PR submission checklist:
